### ISSUE TITLE
Is GitNote open source / how to contribute

### ISSUE THREAD

**Issue by marnovo** [zhaopengme/gitnote#315]
Hi @zhaopengme,

Is GitNote supposed to be open-source (or was it, or will it be)?

If yes, where does the codebase for the application and the documentation live? I assume this repository is just for issues?

I am interested in the project and would like to contribute but haven't found those, nor any guidelines on how to do so.

Thanks!

**Comment by rockandska** [comment]
Had the same questions too

Will be neat to participate to this neat taking app

**Comment by dcsan** [comment]
yeah it's free but closed source and all the issues are in chinese... not a recipe for confidence.
